ci-operator/config/openshift/configmap-reload/master.json

```diff
@@ -0,0 +1,38 @@
+{
+  "tag_specification": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "origin-v3.11",
+    "tag": "",
+    "tag_overrides": {}
+  },
+  "base_images": {
+    "base": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "origin-v3.11",
+      "tag": "base"
+    }
+  },
+  "test_base_image": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "release",
+    "tag": "golang-1.10"
+  },
+  "canonical_go_repository": "github.com/openshift/configmap-reload",
+
+  "images": [
+    {
+      "from": "base",
+      "to": "configmap-reload"
```

Comment:
Dockerfile is failing with 

```
2018/07/18 13:47:17 Build configmap-reload failed, printing logs:
Pulling image "docker-registry.default.svc:5000/ci-op-x1rsxbjp/pipeline@sha256:6c89cd9b8b84f739599ac319f2b28dc79188479e8516c079b6215842375c5f38" ...

Pulling image docker-registry.default.svc:5000/ci-op-x1rsxbjp/pipeline@sha256:827197efeea3e04b6af492b4d822852ae63c58eeaef6406279a5869aab60e47c ...
--> FROM docker-registry.default.svc:5000/ci-op-x1rsxbjp/pipeline@sha256:827197efeea3e04b6af492b4d822852ae63c58eeaef6406279a5869aab60e47c as 0
--> COPY out/configmap-reload /configmap-reload
error: build error: stat /tmp/build/inputs/out/configmap-reload: no such file or directory
2018/07/18 13:47:17 Ran for 33s
```

Is this missing a `dockerfile_path` statement?

Reply:
The `configmap-reload` repository has a [`Dockerfile`](https://github.com/openshift/configmap-reload/blob/openshift-master/Dockerfile) at its root.

Does the system default to `./Dockerfile` as the `dockerfile_path`? If not, I will update this and the other `master.json` files.

Reply:
Looking at this again, this seems to me, that it is using the `Dockerfile` in `master` instead of `openshift-master`. `openshift-master` is set in ` cluster/ci/config/prow/config.yaml`. Do I need to specify the branch in the `master.json` file as well, @smarterclayton?

Reply:
Yeah, this was the different branch thing.  I really, really, really would prefer if master in openshift/* repos meant the same thing across all of them.  Since these are already forks, it'll just make everyone else's life easier.